l10n_ro_report_trial_balance/wizards/trial_balance_wizard.py

```diff
@@ -0,0 +1,101 @@
+# Copyright (C) 2018 FOREST AND BIOMASS ROMANIA SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import models, fields, api
+from odoo.tools.safe_eval import safe_eval
+from odoo.tools import pycompat
+
+
+class RomaniaTrialBalanceReportWizard(models.TransientModel):
+    """Trial balance report wizard."""
+
+    _name = "l10n.ro.report.trial.balance.wizard"
+    _description = "Romania Trial Balance Report Wizard"
+
+    company_id = fields.Many2one(
+        comodel_name='res.company',
+        default=lambda self: self.env.user.company_id,
+        string='Company'
+    )
+    date_range_id = fields.Many2one(
+        comodel_name='date.range',
+        string='Date range'
+    )
+    date_from = fields.Date(required=True)
+    date_to = fields.Date(required=True)
+    target_move = fields.Selection([('posted', 'All Posted Entries'),
+                                    ('all', 'All Entries')],
+                                   string='Target Moves',
+                                   required=True,
+                                   default='posted')
+    hide_account_balance_at_0 = fields.Boolean(
+        string='Hide account ending balance at 0',
+        help='Use this filter to hide accounts '
+             'with an ending balance at 0. '
+             'If all accounts from an account group have ending balance at 0, '
+             'the group will not be printed in the report.')
+    with_special_accounts = fields.Boolean(
+        'With Special Accounts',
+        help="Check this if you want to print classes 8 and 9 of accounts.")
+    account_ids = fields.Many2many(
+        comodel_name='account.account',
+        string='Filter accounts',
+    )
+
+    @api.onchange('date_range_id')
+    def onchange_date_range_id(self):
+        """Handle date range change."""
+        self.date_from = self.date_range_id.date_start
+        self.date_to = self.date_range_id.date_end
+
+    @api.multi
+    def button_export_html(self):
+        self.ensure_one()
+        action = self.env.ref(
+            'l10n_ro_report_trial_balance.l10n_ro_report_trial_balance')
```

Comment:
action = self.env.ref( 'l10n_ro_report_trial_balance.action_l10n_ro_report_trial_balance')
Correct is action_l10n_ro_report_trial_balance

Reply:
Nu prea are importanta, daca nu e setat ia el implicit numele meodului din care face parte, daca da ia toata calea..

Reply:
Eu zic ca e gresit numele actiunii.